### PR TITLE
Doc 314 tutorial pausing a session with cron

### PR DESCRIPTION
**Pull request by sunitaprakash**
# What does this PR do?
Add to the docs Github Action pause
Explain purpose of PR and any useful context.

### CODE REVIEW

File: docs/features/sessions/pause.md [hunk 1]
```diff
@@ -41,3 +41,29 @@ This is so that you don't lose your work!
...
+## Pause a Session Using a Github Action
+
+When you are done with your work day, you can setup Github Action to pause your session.  
+
+Below is scheduled-SessionPause.yml:
+
+```
+on:
+  schedule:
+    # Runs "at 8:25pm" everyday
```
Comment (oojo12) [inline docs/features/sessions/pause.md:54]: update this to contain the link to crontab - https://crontab.guru/
It will be useful for users to easily be able to update the crontab to meet their desired schedule

File: docs/features/sessions/pause.md [hunk 2]
```diff
@@ -41,3 +41,29 @@ This is so that you don't lose your work!
...
+      - uses: actions/checkout@v2
+      - uses: gridai-actions/gridai-login@v0
+        with:
+          gridai-username: ${{ secrets.GRIDAI_USERNAME }} 
+          gridai-key: ${{ secrets.GRIDAI_KEY }}
+      - run: |
+          grid session pause ${{ secrets.SESSION_NAME }} 
+  ```
+
+For example look at the [Github repository](https://github.com/sunitaprakash/grid-session-env).
```
Comment (oojo12) [inline docs/features/sessions/pause.md:69]: Isn't this the exact example taken from your repo? If so why not just link to your repo instead of putting the code in line so that the community is always directed to the most updated example?

**Review by oojo12 — CHANGES_REQUESTED** [review]

**Comment by essiequoi** [comment]
hey @sunitaprakash still need a review from me now or do you still have to update with femi's suggestions?

**Comment by sunitaprakash** [comment]
I will update it with Femi review then ask review from you again 

Thanks

File: docs/features/sessions/pause.md (hunk 1 above)
Comment (sunitaprakash) [reply docs/features/sessions/pause.md]: removed the example

File: docs/features/sessions/pause.md (hunk 2 above)
Comment (sunitaprakash) [reply docs/features/sessions/pause.md:48]: removed the example just pointed it to the Github repo

**Review by oojo12 — APPROVED** [review]

File: docs/features/sessions/pause.md [hunk 3]
```diff
@@ -41,3 +41,8 @@ This is so that you don't lose your work!
 When you have added credits, just resume the session and carry on with your work. The left navigation bar always shows credits remaining.
 
 ![](/images/sessions/autopausing.gif)
+
+## Pause a Session Using a Github Action
+
+When you are not using your session, You can pause it using the Github Action.
```
Comment (oojo12) [inline docs/features/sessions/pause.md:47]: This language could be better. Approved none the less.

For example this reads better:

"We provide an example GitHub Action [template](https://github.com/sunitaprakash/grid-session-env) for pausing Sessions"

Comment (oojo12) [reply docs/features/sessions/pause.md:47]: I will be making the language change then merging.

File: docusaurus.config.js [hunk 4]
```diff
@@ -56,10 +56,6 @@ const config = {
             to:"/features/datastores",
             from:"/products/add-data-to-grid-datastores"
           },
-          {
```
Comment (pritamsoni-hsr) [inline docusaurus.config.js:59]: why is this removed?

**Review by pritamsoni-hsr — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 4 above)
Comment (pritamsoni-hsr) [reply docusaurus.config.js:59]: let's keep it, you can change the ids to new ids

**Comment by pritamsoni-hsr** [comment]
LGTM, rebase without docusaurus.config.js, it should do the job

**Comment by essiequoi** [comment]
closing this and creating a new one because it's too outdated
